addons/zylann.scatter/tools/plugin.gd

```diff
@@ -149,7 +150,7 @@ func _physics_process(delta):
 			action = ACTION_ERASE
 	
 	if _mouse_pressed:
-		if action == ACTION_PAINT:
+		if action == ACTION_PAINT and selected_patterns.size() > 0:
```

Comment:
There is a `if _patterns.size() == 0:` at the beginning of the function, which meant "is any pattern selected". But with your changes its meaning changed inadequately. You should probably fix it rather than adding extra checks down here, because they would be unnecessary